react/src/lib/components/DeckGLMap/layers/layersDefaultProps.ts

```diff
@@ -90,14 +91,15 @@ export const layersDefaultProps: Record<string, unknown> = {
         outline: true,
         logRadius: 10,
         logCurves: true,
-        refine: true,
+        refine: false,
```

Comment:
Why this change?

Reply:
The wells are extremely slow using refine and there are no visible change when turning it of